src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -2004,6 +2031,14 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
             ));
         }
     });*/
+    
+    @computed get selectedGene():Gene {
```

Comment:
since this is specifically for treatment profiles, the name should reflect that. e.g. `selectedGeneForTreatment`